packages/field-base/src/shadow-focus-mixin.js

```diff
@@ -61,15 +58,13 @@ export const ShadowFocusMixin = (superClass) =>
       if (!this.disabled && this.focusElement) {
         const path = event.composedPath();
 
-        // When focus moves from outside and not with Shift + Tab, delegate it to focusElement.
+        // When focus moves to the host element itself, then delegate it to the focusElement
         // This should only move focus when using keyboard navigation, for clicks we don't want to interfere,
         // for example when the user tries to select some text
-        if (this._keyboardActive && path[0] === this && !this.contains(event.relatedTarget) && !this._isShiftTabbing) {
```

Comment:
As mentioned above `_isShiftTabbing` seems unnecessary.

`!this.contains(event.relatedTarget)` also doesn't seem to make a difference.